tldw_Server_API/app/services/stripe_metering_service.py

```
@@ -6,19 +6,374 @@
     """Reconciles local usage tracking with Stripe metering."""
 
     def __init__(self) -> None:
         self._enabled = os.getenv("BILLING_ENABLED", "false").lower() in ("1", "true")
         self._stripe_key = os.getenv("STRIPE_API_KEY", "")
+        self._meter_event_name = os.getenv("STRIPE_METER_EVENT_NAME", "api_requests")
+
+    # ------------------------------------------------------------------
+    # Internal helpers
+    # ------------------------------------------------------------------
+
+    async def _get_db_pool(self) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_get_db_pool`

(ANN401)